platform/android/export/export_plugin.cpp

```diff
@@ -2911,6 +2953,9 @@ Error EditorExportPlatformAndroid::export_project_helper(const Ref<EditorExportP
 		print_verbose("Storing command line flags...");
 		store_file_at_path(assets_directory + "/_cl_", command_line_flags);
 
+		print_verbose("Updating JAVA_HOME environment to " + java_sdk_path);
+		OS::get_singleton()->set_environment("JAVA_HOME", java_sdk_path); //set and overwrite if required
```

Comment:
```suggestion
		OS::get_singleton()->set_environment("JAVA_HOME", java_sdk_path); // Set and overwrite if required.
```
And likely the same treatment for the comment it was copied from. But maybe repeating these comments isn't so useful and they could be removed (it's not clear what the `if required` means here).